lib/src/utils/json_helper.dart

```diff
@@ -408,6 +408,12 @@ class JsonHelper {
   static DateTime? nullableStringTimestampToDate(dynamic json) =>
       json == null ? null : stringTimestampToDate(json);
 
+  static String dateToStringTimestamp(DateTime dateTime) =>
+      dateTime.toIso8601String();
+
+  static String? nullableDateToStringTimestamp(DateTime? dateTime) =>
+      dateTime == null ? null : dateToStringTimestamp(dateTime);
+
```

Comment:
Looks very similar to what is already coded.
If your methods are really useful, that means that parts of the existing code is wrong - which is of course possible.
Would you please check that we really need your methods, like, what if we transform an object into a json into an object, is the datetime kept?
```dart
  /// Returns a [DateTime] from a JSON-encoded String (e.g. '2021-10-29T11:00:56.177379')
  static DateTime stringTimestampToDate(dynamic json) =>
      DateTime.parse(json as String);

  /// Returns a [DateTime] from a JSON-encoded String (e.g. '2021-10-29T11:00:56.177379')
  static DateTime? nullableStringTimestampToDate(dynamic json) =>
      json == null ? null : stringTimestampToDate(json);
```